repository/model/project_invitation.go

```diff
@@ -0,0 +1,69 @@
+package model
+
+import (
+	"time"
+
+	svcmodel "release-manager/service/model"
+
+	"github.com/google/uuid"
+)
+
+type ProjectInvitation struct {
+	ID            uuid.UUID `json:"id"`
+	ProjectID     uuid.UUID `json:"project_id"`
+	Email         string    `json:"email"`
+	ProjectRole   string    `json:"project_role"`
+	Status        string    `json:"status"`
+	TokenHash     []byte    `json:"token_hash"`
+	InviterUserID uuid.UUID `json:"invited_by"`
+	CreatedAt     time.Time `json:"created_at"`
+	UpdatedAt     time.Time `json:"updated_at"`
+}
+
+type ProjectInvitationUpdate struct {
```

Comment:
In the repository layer, I believe `ProjectInvitationUpdate` fits better than `UpdateProjectInvitationInput`.

Reply:
Maybe. If I commented on this and voted for `_Input` name, it was a bit on the edge, so OK. 👍